editor/gui/output_dock.h

```diff
@@ -31,4 +31,12 @@ class OutputDock
 
 	OutputDockSettings& getSettings() { return m_OutputDockSettings; }
 	void setActive(bool enabled) { m_OutputDockSettings.m_IsActive = enabled; }
+
+	enum class OutputState : unsigned int
+	{
+		All = 0,
+		Warning = 1,
+		Error = 2,
+		StateEnd = 3
+	};
```

Comment:
Definitions like these would normally be the first thing in the class

Reply:
Fixed